src/main/java/com/dao/LocalJsonMailDAO.java

```diff
@@ -0,0 +1,97 @@
+package com.dao;
+
+import com.dto.MailDto;
+import com.model.Mail;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
+import org.json.simple.parser.JSONParser;
+import org.json.simple.parser.ParseException;
+import org.springframework.stereotype.Repository;
+
+import javax.annotation.PostConstruct;
+import javax.annotation.PreDestroy;
+import java.io.*;
```

Comment:
Avoid using wilcard imports. You will have to update your IDEA settings to prevent this from happening